### PR TITLE
Config

### PR DESCRIPTION
**Pull request by greenw0lf**
Linked to #6 
- Added settings for sample rate, audio file extension, and whether to convert to mono or not
- Copied and slightly adapted files from [dane-example-worker](https://github.com/beeldengeluid/dane-example-worker)

### CODE REVIEW

**Comment by greenw0lf** [comment]
Might be intentional, but there's `black` where a `line-length` is defined, whereas for `flake8` the default value is not changed so I assume flake8 is not used for lining. However, flake8 fails the pipeline because some lines (from the code that was copied from dane-example-worker) are too long. This needs discussion

**Comment by greenw0lf** [comment]
Ok, I addressed the pipeline fail (and too many errors that were showing up) by adding `.flake8` from [dane-visual-feature-extraction-worker](https://github.com/beeldengeluid/dane-visual-feature-extraction-worker/blob/main/.flake8).

**Review by Kathleendeboer — APPROVED** [review]
Looks good!

File: .flake8
Comment (Veldhoen) [inline .flake8:1]: Ahh yes, good point of adding the .flake8. 
Could you perhaps create a PR to add it to the example-worker too, @greenw0lf?

**Review by Veldhoen — APPROVED** [review]
Looks good! I added a suggestion and a request, but those are not blocking a PR merge.

**Pull request merged**